protocol-specification.md

```diff
@@ -6,7 +6,7 @@ In this chapter, we outline the specification for the abstract LSAT HTTP and gRP
 
 ## Specification
 
-This section defines the "LSAT" authentication scheme, which transmits credentials as `<macaroon(s)>:<preimage>` pairs, where the preimage and the macaroon are each encoded as base64. Multiple macaroons are base64 encoded individually and listed comma separated before the column.  
+This section defines the "LSAT" authentication scheme, which transmits credentials as `<macaroon(s)>:<preimage>` pairs, where the preimage is encoded as hex and the macaroon is encoded as base64. Multiple macaroons are base64 encoded individually and listed comma separated before the column.  
```

Comment:
Does our implementation already support handling multiple LSATs? Also, what's the column in `list comma separated before the column`?

Reply:
No, our implementation doesn't support that yet. But it's required for the discharge macaroons when using 3rd party caveats. So I wanted to have this in the spec to be forward compatible.

Oops, that's supposed to say `colon` not `column`. Fixed.